src/views/SslList.vue

```diff
@@ -498,9 +518,12 @@ export default defineComponent({
       this.setLoadingDocStatus(false)
     },
 
-    async deleteProfile(id: string) {
-      this.clickedRow = id
+    // TODO: need to add that to DeleteCertificate and send only the attachedApps
+    openDeleteCertWarn(certificateId:string) {
+      this.attachedApps = this.sitesByCertNameMap[certificateId]
```

Comment:
The function can be inline as this is the only place that uses it, something along the lines of this perhaps?

```suggestion
      this.attachedApps = []
      this.sites?.forEach((site: Site) => {
        const certId = site.ssl_certificate
        if (certId) {
          if (!this.attachedApps[certId]) {
            this.attachedApps[certId] = []
          }
          this.attachedApps[certId].push(site.name)
        }
      })
```

Reply:
Ignore that. `this.sitesByCertNameMap` is a computed value not a function